src/Message.php

```diff
@@ -89,6 +89,12 @@ class Message {
   const ICON_TYPE_EMOJI = 'icon_emoji';
 
   /**
+   *
+   * @var integer
+   */
+  const MAX_RETRY_ATTEMPTS = 10;
```

Comment:
Yup, the message class is just an adaptor over the client class. We could potentially have other adaptors too. By default, the client has its own implementation and the idea is that the message class can override it. In case the message class provides this variable, then the client class consumes it. Else, it uses it own